src/hotspot/share/gc/g1/g1ConcurrentRefine.cpp

```diff
@@ -112,6 +113,13 @@ jint G1ConcurrentRefineThreadControl::initialize(G1ConcurrentRefine* cr, uint ma
     }
   }
 
+  if (UsePerfData && os::is_thread_cpu_time_supported()) {
+    EXCEPTION_MARK;
+    _g1_concurrent_refine_threads_cpu_time =
+        PerfDataManager::create_variable(NULL_NS, "g1_conc_refine_thread_time",
```

Comment:
See my general comment about name spaces. The name should be something like `*.cpu_time`.